openmls/src/extensions/codec.rs

```diff
@@ -13,12 +13,13 @@ impl Size for Extension {
         2 /* extension type len */
             + 4 /* u32 len */ +
             match self {
-                Extension::Capabilities(e) => e.tls_serialized_len(),
                 Extension::ApplicationId(e) => e.tls_serialized_len(),
-                Extension::LifeTime(e) => e.tls_serialized_len(),
-                Extension::ParentHash(e) => e.tls_serialized_len(),
                 Extension::RatchetTree(e) => e.tls_serialized_len(),
                 Extension::RequiredCapabilities(e) => e.tls_serialized_len(),
+                Extension::ExternalPub(e) => e.tls_serialized_len(),
+                Extension::Capabilities(e) => e.tls_serialized_len(),
+                Extension::Lifetime(e) => e.tls_serialized_len(),
```

Comment:
The `Lifetime` rename seems unnecessary as it's not an extension anymore.

Reply:
It's still in the code. So let's keep the rename for now and delete later.